### PR TITLE
Readme for warehouse

### PR DESCRIPTION
**Pull request by sravfeyn**
@calellowitz @gcapalbo

### CODE REVIEW

File: corehq/warehouse/README.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+```
+
+Use `commit_table <data_model_slug> <batch_id>` to load data of data_model along with any of its dependencies' data for duration specified by the batch_id. For e.g.
+
+```
+./manage.py commit_table user_staging 222617b9-8cf0-40a2-8462-7f872e1f1344
+```
+
+
+To flush staging data of a particular batch use `./manage.py clear_staging_records`. During the development, you could use `DROP TABLE` (or `./manage.py migrate warehouse zero`) sql commands to clear fact/dimension data.
```
Comment (gcapalbo) [inline corehq/warehouse/README.md]: for development, i think using truncate would be easier than dropping and recreating the tables

File: corehq/warehouse/README.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+## Commands
+
+Use `create_batch` to create a batch. This just creates a new batch record.
+
+```
+./manage.py create_batch 222617b9-8cf0-40a2-8462-7f872e1f1344 -s 2012-05-05 -e 2018-06-05
+```
+
+Use `commit_table <data_model_slug> <batch_id>` to load data of data_model along with any of its dependencies' data for duration specified by the batch_id. For e.g.
```
Comment (gcapalbo) [inline corehq/warehouse/README.md]: it sounded like based on our side chat that it doesn't load the dependencies' data right?

Comment (sravfeyn) [reply corehq/warehouse/README.md]: Yeah, sorry. I had updated this, but didn't push. See now

File: corehq/warehouse/README.md (hunk 1 above)
Comment (sravfeyn) [reply corehq/warehouse/README.md]: Agree, updated.

**Review by gcapalbo — APPROVED** [review]

**Pull request merged**